index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portfolio</title>
+    <script src="https://kit.fontawesome.com/6b55e5dec8.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <!-- this is the navigation header -->
+    <header class="navigation">
+        <div class="nav-logo">
+            <a href="">WELCOME</a>
+        </div>
+        <nav class="toggle"><button><i class="fa-solid fa-bars"></i></button></nav>
+    </header>
+
+    <!-- this is the section part -->
+
+    <section class="first-section">
```

Comment:
There is a little issue regarding the position of the background on this headline section 😥 Please try to fix it so that your design resembles the positioning of the background elements as it's displayed in the Figma design 🤓 Kindly make sure that every text has the same font like the ones used in Figma. 


Your design 
![Captura2](https://user-images.githubusercontent.com/15807343/153900397-db7bfa4b-865f-4c44-8701-3f691397ffdb.PNG)

Figma's
![Captura](https://user-images.githubusercontent.com/15807343/153900334-ff7e1a0b-3c07-4d4c-a96f-ea369a92e452.PNG)